scanners/zap/zap.py

```diff
@@ -203,6 +203,12 @@ def _get_standard_options(self):
             ["-config", f"network.localServers.mainProxy.port={local_port}"]
         )
 
+        override_cfg = self.my_conf("miscOptions.overrideConfigs")
+        if override_cfg:
+            for cfgitem in override_cfg:
```

Comment:
probably adding a try/except might be needed here, in case the user does not provide a list.
Either a hard error, or a warning, but the user should at least be warned that this entry should be a list.

Reply:
thanks. updated :)